features/user_can_create_invoice.feature

```diff
@@ -0,0 +1,13 @@
+Feature: User can create invoice
+
+  Background:
+    Given the following user exist
+      | email         | password |
+      | user@user.com | password |
+
+    And I am logged in as "user@user.com"
+    And I am on the invoices page
+
+  Scenario: User can create an invoice
+    When I click on "Create new invoice"
+    Then I should see "Invoice number:"
```

Comment:
I would expand on this scenario. But I assume this is a WIP.

Reply:
Yeah felt like this story/PR was getting pretty big as it it.